### DISCUSSION TITLE
Route Rules - How to use /** except /api/** routes?

### DISCUSSION THREAD

**Discussion by SleepiestAdam** [nuxt/nuxt#16048] [Questions]
I'm trying to essentially setup the following route rules whereby it doesn't statically render any API routes, but does statically render everything else.

Unfortunately I'm running into an issue where it's treating the API as static and all subsequent requests once an API call is made don't actually trigger the API route, they just return whatever result was returned the first time.

```
routeRules: {
    '/api/**': { cors: true },
    '/**': { swr: 86400 }
}
```

If I remove the following line then the API routes work:

`'/**': { swr: 86400 }`

So I've broadly narrowed it down to a cacheing issue. I gather the /** is therefore superseding the /api/** rule.

My question is - is there any way to set "all but the /api/** routes" as { swr: 86400 } without having to list out basically every non api route?

**Comment by TheAlexLichter** [comment]
Could you specify `swr: false` for `/api/**`?

**Comment by kiritowch** [comment]
这是一封自动回复邮件。已经收到您的来信，我会尽快回复。
